app/assets/stylesheets/scaffolds.scss

```diff
@@ -3,18 +3,6 @@
 @import 'variables';
 @import 'mixins';
 
```

Comment:
This scaffolds.scss file is kind of a dump where sometimes very important styling used to be put... I move some of it to more appropriate files.